src/components/about/AboutHeaderImage.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Container, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(theme => {
+  return {
+    imageContainer: image => ({
+      width: "100%",
+      height: 373,
+      backgroundImage: image.url,
+      backgroundPosition: "center",
+      backgroundSize: "cover",
+      marginBottom: theme.spacing(2.5),
+      display: "flex",
+      flexDirection: "column",
+      justifyContent: "center",
+      textAlign: "center"
+    }),
+    headlineText: {
+      color: "white",
```

Comment:
This white text was hard to read on the image. Could we maybe add a drop shadow or something to make it easier to read? The `textShadow` attribute might help.

Alternatively, I'd be okay to remove this entire header because "Climate Connect" is already in the site's header.

Reply:
I agree that in terms of text this header is not necessary at all, but a picture on the top of the page is visually appealing. I would keep it for now. I added textShadow to the text.